Waypoint[] waypointArrayIn
float64 current_x
float64 current_y
float64 current_heading
uint8 collision
float32 collisionDistance
---
Waypoint[] waypointArrayOut
